raiden/network/pathfinding.py

```diff
@@ -587,6 +587,8 @@ def query_paths(
             )
         except ServiceRequestIOURejected as error:
             code = error.error_code
+            log.debug("Pathfinding Service rejected IOU", error=error)
```

Comment:
Does `error` tell structlog that this is an exception? Otherwise you can do that somehow (was it `exc_info=True`?). But just having the error message is probably enough here, too.

Reply:
No, but I think it's better to not have the full stack trace printed.